.github/CODEOWNERS

```diff
@@ -645,6 +645,9 @@
 # ServiceLabel: %Network - Private Link %Service Attention
 #/<NotInRepo>/                                                     @privlinksuppgithub
 
+# PRLabel: %Nginx
+/sdk/nginx/														   @sofwiti @limgu @yashi
```

Comment:
None of these folks are eligible to be CODEOWNERS, as they do not have write access to the repository.   Please follow the Azure SDK onboarding guide and ensure that they've requested write access, have joined the Microsoft and Azure orgs, and those memberships are flagged as public.

Reply:
I have updated the CODEOWNERS with the right GitHub accounts